lib/h264.cpp

```diff
@@ -479,7 +480,13 @@ namespace h264{
       log2MaxPicOrderCntLsbMinus4 = bs.getUExpGolomb();
       derived_maxPicOrderCntLsb = pow(2, log2MaxPicOrderCntLsbMinus4 + 4);
     }else if (picOrderCntType == 1){
-      WARN_MSG("picOrderCntType == 1 encountered, which has not-yet implemented parameters");
+      deltaPicOrderAlwaysZeroFlag = bs.get(1);
+      bs.getExpGolomb(); // offset_for_non_ref_pic
+      bs.getExpGolomb(); // offset_for_top_to_bottom_field
+      numRefFramesInPicOrderCntCycle = bs.getUExpGolomb();
+      for (size_t i = 0; i < numRefFramesInPicOrderCntCycle; i++){
+        bs.getExpGolomb(); // offset_for_ref_frame[i]
+      }
       return;
```

Comment:
I am an idiot - this "return" should get removed. Obviously, it was there with the WARN_MSG. I accidentally removed this file during formatting and had to recreate changes from older backup. Forgot about this one :-(